react/country/IDN.ts

```diff
@@ -0,0 +1,155 @@
+import { POSTAL_CODE } from '../constants'
+
+export default {
```

Comment:
can you add the typings here please?
```suggestion
const rules: PostalCodeRules = {
```

and in the end of the file you can export this variable

```ts
export default rules
```

we recently changed this to add better typescript coverage and safety on this repo. I was going to create a post on #checkout-localization with a FYI, motivation, etc, but I saw that the other country files in this PR were already in this new format 😄 (kudos for that @kaio-donadelli and @sheilagomes)

Reply:
Done!